sklearn/feature_selection/_rfe.py

```diff
@@ -195,21 +212,23 @@ def _fit(self, X, y, step_score=None):
 
             estimator.fit(X[:, features], y)
 
-            # Get coefs
-            if hasattr(estimator, 'coef_'):
-                coefs = estimator.coef_
-            else:
-                coefs = getattr(estimator, 'feature_importances_', None)
-            if coefs is None:
-                raise RuntimeError('The classifier does not expose '
-                                   '"coef_" or "feature_importances_" '
-                                   'attributes')
+            # Get feature importance
+            importance_getter = self.importance_getter
```

Comment:
The following block with part of `_get_feature_importance`, we should avoid that